src/scss/components/common/SelectField.scss

```diff
@@ -0,0 +1,49 @@
+@import '../../base/variables.scss';
+@import '../../base/typography.scss';
+
+.selectWrapper {
+    display: flex;
+    flex-direction: column;
+    label {
+      text-align: center;
+      text-transform: uppercase;
+      font-size: 0.688rem;
+      margin: 0.25rem 0;
+      line-height: 0.813rem;
+      letter-spacing: 0.103rem;
+      font-weight: 600;
+      &.error {
+        color: $error;
+      }
+    };
+    select {
+      @include font-style;
+      -webkit-appearance: none;
+      -moz-appearance: none;
+      text-indent: 1px;
+      letter-spacing: 0.075rem;
+      padding: 0.75rem 1.25rem;
+      margin: 0.5rem 0 0 0;
+      border: 0.063rem solid $black;
+      border-radius: 0;
+      box-sizing: border-box;
+      outline: none;
+      &.error {
+        border-color: $error;
+      }
+      &:focus {
+        outline-style: none;
+        box-shadow: none;
+      }
+    }
+    span {
```

Comment:
if span will only be used for errors, it might be a good idea to use a class instead